samples/nrf9160/modem_shell/README.rst

```diff
@@ -499,6 +499,29 @@ Examples
 
      rest -d example.com -l 1024 -m head -H "X-foo1: bar1\x0D\x0A" -H "X-foo2: bar2\x0D\x0A"
 
+----
+
+Cloud
```

Comment:
We are missing compilation instructions from `Building and running` section so there should be a new subsection like for other overlays.

Reply:
Done!